### ISSUE TITLE
The training code of Music2Latent2

### ISSUE THREAD

**Issue by Alidaling** [SonyCSLParis/music2latent#6]
This is a novel piece of work. We have trained a model using our own data based on your code, and the results are reliable! I would like to ask if you could also open-source the code of Music2Latent2, because Music2Latent2 has better performance. We are looking forward to your reply.

**Comment by stefanlattner** [comment]
Yes, we plan to publish newer versions of music2latent soon. However, we cannot yet give reliable estimates when this will happen.

**Comment by ariepavlov** [comment]
Hey, I was able to reproduce Music2Latent2 from the paper and it works great for mono, but when im trying to train stereo, i get mode collapse. Basically, after the transformer layers of the encoder the tanh normalisation gets saturated meaning that the K semantic representations are all either 1 or -1. Were you able to train stereo? or have you ran into this problem?

**Comment by happyTonakai** [comment]
> Hey, I was able to reproduce Music2Latent2 from the paper and it works great for mono, but when im trying to train stereo, i get mode collapse. Basically, after the transformer layers of the encoder the tanh normalisation gets saturated meaning that the K semantic representations are all either 1 or -1. Were you able to train stereo? or have you ran into this problem?

That's cool. Could you share your code?
